cmd/controller-manager/app/controller-manager.go

```diff
@@ -117,6 +130,14 @@ func Run(opts *options.Options) error {
 		klog.Fatalf("Invalid Feature Gate: %v", err)
 	}
 
+	if len(masterURL) == 0 && len(kubeconfig) == 0 && len(opts.Config.FederationNamespace) == 0 {
+		fedName, err := defaultNamespace()
```

Comment:
`s/fedName/fedNamespace/`

Please use clear and unambiguous names.  Avoid brief names that are insufficiently descriptive.